src/MurmurClient.ts

```diff
@@ -82,7 +88,7 @@ export class MurmurClient {
     }
     const request: NewRequest = {
       validity,
-      current_block: (await this.getCurrentBlock()).toNumber(),
+      current_block: this.finalizedBlockNumber,
```

Comment:
can't this value be 0? if so we should implement an async function that await for this value to be > 0